framework/src/executioners/Eigenvalue.C

```diff
@@ -71,3 +76,24 @@ Eigenvalue::execute()
 
   Steady::execute();
 }
+
+void
+Eigenvalue::postSolve()
+{
+  Real val = getParam<Real>("normal_factor");
```

Comment:
Looks to me users can set normal_factor without giving the postprocessor name. In such a case, the value will be used for scaling directly. This line needs to be moved into the following if branch. A separate scaling factor is declared here initialized to 1.